legal-api/src/legal_api/services/filings/validations/continuation_in.py

```diff
@@ -115,7 +116,10 @@ def _validate_foreign_jurisdiction(filing_json: dict, filing_type: str, legal_ty
     """Validate continuation in foreign jurisdiction."""
     msg = []
     foreign_jurisdiction = filing_json['filing'][filing_type]['foreignJurisdiction']
+    incorporation_date = filing_json['filing'][filing_type]['foreignJurisdiction']['incorporationDate']
     foreign_jurisdiction_path = f'/filing/{filing_type}/foreignJurisdiction'
```

Comment:
No need to check if incorporationDate exists, because it's mandatory from UI

Reply:
On the contrary, if it's mandatory in the UI then the Legal API should also make sure it's mandatory (since we support "API users" who call our endpoints directly -- our endpoints are not called exclusively by our UIs).

APIs should always validate the incoming data to prevent data issues.

Reply:
its required as per schema. no need to check if incorporationDate exist in this validation

Reply:
We don't need an update now, right? Is this PR ready to go?

Reply:
yes